sdk/storage/storage-queue/README.md

```diff
@@ -1,30 +1,42 @@
-# Azure Storage client library for JavaScript - Queue
+# Azure Queue Storage client library for JavaScript
```

Comment:
I think "Azure Storage Queue" reads better to me, and maps more closely with the package name.

Reply:
After discussing we resolved to use "Azure Storage Queue." For posterity, the reason I changed it to Queue Storage is because that is how the Service documentation refers to it. Azure Storage Queue more closely mirrors our package name. We can change down the line if we need to.